tools/package-windows-x64.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# This script is used to package the macOS x64 binaries for sshnoports.
+# It first unpacks the prebuilt binaries, then it calls the notarize-macos.sh
+# script to notarize the binaries
+
+FULL_PATH_TO_SCRIPT="$(realpath "${BASH_SOURCE[0]}")"
+SCRIPT_DIRECTORY="$(dirname "$FULL_PATH_TO_SCRIPT")"
+ROOT_DIRECTORY="$SCRIPT_DIRECTORY/.."
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <tgz file>"
+  exit 1
+fi
+
+if [ "$(uname)" != "Darwin" ]; then
```

Comment:
Why we're running a Windows related script on macOS deserves some explanation

Should we have something that runs on Windows for anybody doing this for themselves?

Reply:
This script is only for packaging the release, if they want to build from source, they don't even need this step.